### PR TITLE
Implement tinyMCE as text editor.

### PR DESCRIPTION
**Pull request by bengreeley**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Closes #2

File: php/class-integration.php [hunk 1]
```diff
@@ -52,5 +52,8 @@ public function enqueue_scripts() {
 
 		wp_register_style( 'scui_richtext-summernote-css', $this->plugin->dir_url . 'lib/summernote/css/summernote.min.css' );
```
Comment (mehigh) [inline php/class-integration.php:53]: All summernote assets will be discarded.

File: css/shortcode-ui-richtext.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* Make sure any popup boxes appear */
+div.mce-inline-toolbar-grp.mce-arrow-up,
+div.mce-floatpanel[style]{
+    z-index: 999999!important;
```
Comment (mehigh) [inline css/shortcode-ui-richtext.css:4]: I'll try to stay away from !important rules, and see whether a stronger selector would suffice here.

**Review by mehigh — APPROVED** [review]
Thank you for your contribution. I'll make a few changes myself in this new branch before merging into master. I tested it and it looks good.

**Comment by szepeviktor** [comment]
@mehigh How about your two comments?

**Comment by mehigh** [comment]
@szepeviktor I've merged this into a feature branch instead of master to address those items myself. It's small stuff anyway.

**Comment by szepeviktor** [comment]
OK
https://github.com/xwp/wp-shortcode-ui-richtext/commits/feature/tinymce-editor

**Pull request merged**